blueprints/data-solutions/vertex-mlops/variables.tf

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "bucket_name" {
+  description = "GCS bucket name to store the Vertex AI artifacts."
+  type        = string
+  default     = null
+}
+
+variable "dataset_name" {
+  description = "BigQuery Dataset to store the training data."
+  type        = string
+  default     = null
+}
+
+
+variable "group_iam" {
+  description = "Authoritative IAM binding for organization groups, in {GROUP_EMAIL => [ROLES]} format."
```

Comment:
This is for projects role, not for organization

Reply:
Good catch!